.github/workflows/ci.yaml

```
@@ -33,7 +33,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5 # v5.5.0
         with:
-          go-version: "~1.24"
+          go-version: "~1.25.0"
```

reverting this for now
```suggestion
          go-version: "~1.24"
```